### ISSUE TITLE
-ac improvement

### ISSUE THREAD

**Issue by Sh1Yo** [ffuf/ffuf#189]
Sometimes I can see that an application can respond with `words:131 lines:21` when a path does not exist and `words:129 lines:21` when a path exists. But as I can see the `-ac` option will filter `words:131 lines:21` and a potential path will be lost. I think it would be better to filter only one of `size/words/lines` in the case if something of them is stable.

**Comment by joohoi** [comment]
I agree and as such, the current autocalibration functionality leaves much to desire. I'm in the process of doing complete rewrite on this.
